test/test.dart

```diff
@@ -710,3 +710,16 @@ Future<void> enableAllAutomaticRecovery() async {
     await client.setAutomaticRecoveryEnabled(appName, true);
   }
 }
+
+
+extension $WriteExtension on Realm {
```

Comment:
why the `$` here

Reply:
I wanted to hide this extension. But, yes, it is better they to be methods.